PinpointKit/PinpointKit/Sources/Core/MailSender.swift

```diff
@@ -62,13 +62,13 @@ open class MailSender: NSObject, Sender {
     open func send(_ feedback: Feedback, from viewController: UIViewController?) {
         guard let viewController = viewController else { fail(with: .noViewControllerProvided); return }
         
+        self.feedback = feedback
```

Comment:
We should move this above the view controller guard, so that the `.noViewControllerProvided` error will get passed to the appropriate delegates.

Reply:
Good point